### PR TITLE
dev 

### PR DESCRIPTION
**Pull request by blinkinglight**

### CODE REVIEW

File: app/Http/Middleware/ValidateProfile.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Closure(\Illuminate\Http\Request): (\Symfony\Component\HttpFoundation\Response)  $next
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        $model = app(Config::class);
+        if($model->count() === 0 && !$request->routeIs('profile.edit')) {
+            return redirect()->route('profile.edit')->with('status', 'profile-missing');
```
Comment (nycterent) [inline app/Http/Middleware/ValidateProfile.php:21]: should not profile-missing be a lang string?

File: app/Http/Middleware/ValidateProfile.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        }
+        $config = $model->first();
+        if(
+            $config->company_name === null 
+            && $config->invoice_series_deb === null 
+            && $config->invoice_series_cre === null 
+            && $config->invoice_series_pre=== null 
+            && !$request->routeIs('profile.edit')) {
+            return redirect()->route('profile.edit')->with('status', 'profile-missing');
```
Comment (nycterent) [inline app/Http/Middleware/ValidateProfile.php:30]: same as 21

File: resources/views/components/search_buttons.blade.php [hunk 3]
```diff
@@ -11,7 +11,7 @@ class="hidden-xs">Sukurti naują</span></a>
             <label for="id-047724" class="visually-hidden">ieškoti</label>
             <div class="form-regular__wrap">
                 <button class="form-regular__icon"><i class="icon-search" aria-hidden="true"></i></button>
-                <input type="text" name="text" placeholder="Ieškoti kliento" id="id-047724">
+                <input type="text" name="q" placeholder="Ieškoti" id="id-047724">
```
Comment (nycterent) [inline resources/views/components/search_buttons.blade.php:14]: no lang string?

File: resources/views/contrahents/edit.blade.php
Comment (nycterent) [inline resources/views/contrahents/edit.blade.php:1]: lang strings are missing

**Review by nycterent — CHANGES_REQUESTED** [review]

File: .env.ci [hunk 4]
```diff
@@ -1,4 +1,7 @@
+APP_KEY=base64:WHjNafRAPaESNII+k2LXsZOLrh8U0lQ5AeVkh7hm9eg=
```
Comment (nycterent) [inline .env.ci:1]: any security issues that might arise due to this?

File: app/Http/Requests/Auth/LoginRequest.php [hunk 5]
```diff
@@ -29,6 +29,7 @@ public function rules(): array
         return [
             'email' => ['required', 'string', 'email'],
             'password' => ['required', 'string'],
+            // 'g-recaptcha-response' => 'required|recaptchav3:register,0.5'
```
Comment (nycterent) [inline app/Http/Requests/Auth/LoginRequest.php:32]: should go if not used

File: app/Http/Requests/ProfileUpdateRequest.php [hunk 6]
```diff
@@ -16,8 +16,16 @@ class ProfileUpdateRequest extends FormRequest
     public function rules(): array
     {
         return [
-            'name' => ['required', 'string', 'max:255'],
-            'email' => ['required', 'string', 'lowercase', 'email', 'max:255', Rule::unique(User::class)->ignore($this->user()?->id)],
+            "seller_idv" => ["required", Rule::in(["taip", "ne"])],
```
Comment (nycterent) [inline app/Http/Requests/ProfileUpdateRequest.php:19]: no lang string?

**Review by nycterent — CHANGES_REQUESTED** [review]

File: app/Http/Requests/ProfileUpdateRequest.php (hunk 6 above)
Comment (blinkinglight) [reply app/Http/Requests/ProfileUpdateRequest.php:19]: doesnt relates to lang

Comment (nycterent) [reply app/Http/Requests/ProfileUpdateRequest.php:19]: so why in LT?

**Review by nycterent — APPROVED** [review]

**Pull request merged**